projects/components/src/table/table-api.ts

```diff
@@ -25,8 +25,8 @@ export interface TableColumnConfig<TableColumnOptions = unknown> {
   sortable?: boolean;
   filterable?: boolean;
   alignment?: TableCellAlignmentType;
-  width?: number | string;
-  minWidth?: number | string;
+  width?: TableColumnWidth;
```

Comment:
Nitpick: I think it would be a good idea to add some type doc explaining that passing a number without a unit is a flex width, and that `px` and `%` are also supported. Either here (and on the `maxWidth`, or on the `TableColumnWidth` type.

Reply:
+1